src/study-session/session-control.js

```diff
@@ -7,7 +7,7 @@ const { v4: uuidv4 } = require('uuid');
 const log = require('../util/logger');
 const { appData } = require("../api_recievers/influxqueries");
 let winApiThread;
-let sessionId;
+let sessionId = null;
```

Comment:
I think we assume this is undefined by default instead of null, which I think can be treated slightly differently so I'd prefer if we just used undefined here